sdk/bootstrap/main_test.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import "io/ioutil"
+import "log"
+import "os"
+import "testing"
+import "github.com/stretchr/testify/assert"
+
+// Exercise rendering of truthy/falsy values for both regions and variables:
+var templateContent = `dc={{CASSANDRA_LOCATION_DATA_CENTER}}
+{{#PLACEMENT_REFERENCED_ZONE}}
```

Comment:
In the tests, `PLACEMENT_REFERENCED_ZONE` and `FOO` get assigned the same value. Could one then not combine the two blocks here:
```
{{#FOO}}
one={{FOO}}
rack={{ZONE}}
{{/FOO}}
{{^FOO}}
two={{FOO}}
rack=rack1
{{/FOO}}
```